src/systemTest/java/randoop/main/RandoopSystemTest.java

```diff
@@ -617,6 +644,7 @@ public void runContractsTest() {
         new CoverageChecker(
             options,
             "examples.Buggy.BuggyCompareToTransitive.getTwo() ignore",
+            "examples.Buggy.BuggyEqualsTransitive.getTwo() ignore",
```

Comment:
I'm concerned about this change.  Do you understand the reason for it?

Reply:
We decided in a call that this was most likely an outcome of having deterministic random testing.